src/Sylius/Bundle/ThemeBundle/DependencyInjection/Configuration.php

```diff
@@ -27,7 +38,85 @@ public function getConfigTreeBuilder()
         $treeBuilder = new TreeBuilder();
         $rootNode = $treeBuilder->root('sylius_theme');
 
-        $sourcesNodeBuilder = $rootNode->addDefaultsIfNotSet()->fixXmlConfig('source')->children()->arrayNode('sources')->addDefaultsIfNotSet()->children();
+        $this->addResourcesConfiguration($rootNode);
+        $this->addSourcesConfiguration($rootNode);
+
+        return $treeBuilder;
+    }
+
+    /**
+     * @param ArrayNodeDefinition $rootNode
+     */
+    private function addResourcesConfiguration(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->addDefaultsIfNotSet()
+            ->children()
+                ->scalarNode('driver')->defaultValue(SyliusResourceBundle::DRIVER_DOCTRINE_ORM)->end()
+                ->arrayNode('resources')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('theme')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->variableNode('options')->end()
+                                ->arrayNode('classes')
+                                    ->addDefaultsIfNotSet()
+                                    ->children()
+                                        ->scalarNode('model')->defaultValue(Theme::class)->cannotBeEmpty()->end()
+                                        ->scalarNode('interface')->defaultValue(ThemeInterface::class)->cannotBeEmpty()->end()
+                                        ->scalarNode('controller')->defaultValue(ResourceController::class)->cannotBeEmpty()->end()
+                                        ->scalarNode('repository')->defaultValue(ThemeRepository::class)->cannotBeEmpty()->end()
```

Comment:
Repository should not be placed in this configuration. We don't want to couple this with doctrine orm.